block/src/async_io.rs

```diff
@@ -99,4 +99,7 @@ pub trait AsyncIo: Send {
     ) -> AsyncIoResult<()>;
     fn fsync(&mut self, user_data: Option<u64>) -> AsyncIoResult<()>;
     fn next_completed_request(&mut self) -> Option<(u64, i32)>;
+    fn submit_batch_reuests(&mut self) -> AsyncIoResult<()> {
```

Comment:
The commit message summary should be more explicit than "add new function" - "add submit_batch_requests() to AsyncIo" trait